### DISCUSSION TITLE
Error while installing gym ignition

### DISCUSSION THREAD

**Discussion by PravatKumarPanda** [AndrejOrsula/drl_grasping#110] [Q&A]
![image](https://github.com/AndrejOrsula/drl_grasping/assets/74763021/defe94fa-98f5-4b08-bd96-2e0907ce8bdd)
Got this error!

How can i fix it?

**Comment by AndrejOrsula** [comment]
Hello @PravatKumarPanda,

It is difficult to tell with the information you provided, but it seems that [iDynTree](https://github.com/robotology/idyntree) is not installed properly on your system.

Try to follow the instructions in the [`Dockerfile`](https://github.com/AndrejOrsula/drl_grasping/blob/b2283544d19c3b202b14caa03e233ac4598ccee2/Dockerfile) to see if that can resolve your issue.

https://github.com/AndrejOrsula/drl_grasping/blob/b2283544d19c3b202b14caa03e233ac4598ccee2/Dockerfile#L107-L149

---

However, you might be facing other issues if you are trying to install this project locally, as many of the dependencies have been updated, and [Gym-Ignition](https://github.com/robotology/gym-ignition) is unfortunately no longer maintained. You can find further discussion [here](https://github.com/AndrejOrsula/drl_grasping/issues/104#issuecomment-1498837989).
If you want to try out this project, the simpler alternative approach is to use the prebuilt Docker image(s) as documented under [instructions](https://github.com/AndrejOrsula/drl_grasping#instructions).
